x-pack/legacy/plugins/maps/public/connected_components/widget_overlay/attribution_control/_attribution_control.scss

```diff
@@ -4,3 +4,7 @@
   pointer-events: all;
   padding-left: $euiSizeM;
 }
+
+.mapAttributionControl__fullScreen {
+  padding-left: $euiSizeXXL * 4 !important;
```

Comment:
I'm not an expert on this.

Wouldn't `margin-left` work here? The attribution box won't stretch over the exit-full-screen-button then, as opposed to padding, where it does stretch over.

It would also remove the need for the `!important` override, which if I understand it is somewhat of a code-smell in CSS.

```
.mapAttributionControl__fullScreen {
  margin-left: $euiSizeXXL * 4;
}
```